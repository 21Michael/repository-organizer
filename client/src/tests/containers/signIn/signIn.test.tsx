import React from "react";
import { render, cleanup } from "@testing-library/react";
import SignIn from "../../../containers/signIn/signIn";
import "@testing-library/jest-dom/extend-expect";
import { BrowserRouter } from "react-router-dom";
import store from "../../../storeReduxToolkit/configStore";
import { Provider } from "react-redux";
import "mutationobserver-shim";

export default describe("SignIn: ", () => {
  afterEach(() => {
    cleanup();
  });

  test("signIn", () => {
    const { container } = render(
      <Provider store={store}>
        <BrowserRouter>
          <SignIn />
        </BrowserRouter>
      </Provider>
    );
    expect(container).toMatchSnapshot();
  });
});
