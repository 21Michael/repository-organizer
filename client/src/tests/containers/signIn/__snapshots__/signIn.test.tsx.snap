// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignIn:  signIn 1`] = `
<div>
  <div
    class="wrapper"
  >
    <form
      class="form"
    >
      <h1
        class="title"
      >
         
        Authorization
         
      </h1>
      <div
        class="wrapper"
      >
        <label
          class="label"
        >
          Email
          :
          <div
            class="wrapper"
          >
            <input
              autocomplete="email"
              class="input"
              name="email"
              placeholder="email@gmail.com"
              type="email"
              value=""
            />
            <span
              class="input__inform"
            />
          </div>
        </label>
        <label
          class="label"
        >
          Password
          :
          <div
            class="wrapper"
          >
            <input
              autocomplete="current-password"
              class="input"
              name="password"
              placeholder="********"
              type="password"
              value=""
            />
            <span
              class="input__inform"
            />
          </div>
        </label>
      </div>
      <div
        class="wrapper"
      >
        <button
          class="button null"
          name="submit"
          type="submit"
        >
          Sign In
        </button>
        <div
          class="buttonLink signup"
        >
          <a
            class="link"
            href="/signup"
          >
            Sign Up
          </a>
        </div>
      </div>
      <div
        class="wrapper"
      >
        <a
          class="link"
          href="http://localhost:3000/auth/github"
        >
          <span>
            sign with 
            github
          </span>
        </a>
      </div>
    </form>
  </div>
</div>
`;
